openghg/store/base/_datasource.py

```diff
@@ -283,6 +267,74 @@ def get_dataset_daterange_str(self, dataset: Dataset) -> str:
 
         return daterange_str
 
+    def get_representative_daterange_str(self, dataset: Dataset, period: Optional[str] = None) -> str:
+        """
+        Get representative daterange which incorporates any period the data covers.
```

Comment:
This could do with a full docstring.

Reply:
Ok I've added more details for this one.